tests/plugins/test_pull_base_image.py

```diff
@@ -245,3 +247,7 @@ class MockResponse(object):
         runner.run()
 
     assert error_message in exc.value.args[0]
+
+
+def test_pull_scratch_base():
+    test_pull_base_image_plugin('localhost:5000', 'scratch:latest', [], ['scratch:latest'])
```

Comment:
check at least the log message:
 self.log.debug('base image is scratch, do not attempt to pull')